### DISCUSSION TITLE
什麼樣的人適合念電機系?(不正經介紹文)

### DISCUSSION THREAD

**Discussion by HsuSungTing** [1101-GS4719-C/Python-Community-GS4719-C#1201] [General]
(官方講解):適合念電機系的人通常需要擁有以下特質
1.對數學和物理有濃厚的興趣：電機系的課程涉及到很多高等數學和物理學知識，因此對這些學科有興趣的人更容易掌握這些知識。

2.有邏輯思維能力：電機系的課程需要學生具備良好的邏輯思維能力，能夠將複雜的問題分解成更小的部分，並進行分析和解決。

3.具備創新和探索精神：電機系是一個不斷創新和發展的領域，需要學生具備創新和探索精神，勇於開拓新的領域和挑戰自我。

4.具備良好的解決問題能力：電機系的課程需要學生學習如何解決複雜的問題，因此需要具備良好的解決問題能力。

//-------------------不負責任介紹文---------------------------

最近去學校的就業博覽會時，真的發現一個電機系各自表述，真的被各種人資嚇到
我覺得以下這張圖可以表明外人對電機系的誤解:
<img width="600" alt="image" src="https://user-images.githubusercontent.com/89059171/229361847-87612c54-fd98-4708-810e-655215e233a6.png">
<圖源:電機仔迷因>
//------------------------------------------------------------------
就業博覽會的人資:所以電機系真的會寫程式???

**Comment by DuxtinTai** [comment]
當然啦，我們電機系可不是只會用電鍋煮泡麵，還會用程式寫出自動加熱的電鍋呢！畢竟在電機系，我們可是要學會讓電子和程式碼一起跳舞的啊！

**Comment by KevinCheng42** [comment]
我覺得同學分析得十分精確，梗圖也很有趣！
身為電機系的一份子，以前常常也有長輩會問電機系畢業後是不是要當水電工呢~
雖然我們的確會學如何使用三用電錶等儀器～

**Reply by HsuSungTing** [reply]
真的，我填志願的時候想念電機系，但...
我媽跟我說，他覺得我連除濕機有時候都裝不好，所以不適合念電機系.......
我到今天都還在理解這之間的關聯...

**Comment by shuxian12** [comment]
我覺得這個就跟資工系一樣，大家都以為資工系會修電腦，到底誰沒事會修電腦我也是很納悶，我們又不會把主機板猜開來觀察，電腦死當了我們也是雙手開開，直接放棄比較快
但我必須得說我覺得電路板的接線還有三用電錶，我到現在還是看不懂它的原理，每次做完實驗就直接忘ㄌ，respect電機系
💯 💯

**Comment by curtis1017** [comment]
我的系上，有些人並不是真的感興趣，而是想要透過系上的資源，或是拚系牌的方式，來轉系到電機系，我個人的看法是，要真正理解過每個系在做什麼，且是否跟興趣相合後才選系，不要盲目追求大家眼中的第一名。
